docs/release-notes-pxc-v8.0.38.md

```diff
@@ -0,0 +1,3 @@
+# Percona Distribution for MySQL 8.0.38 using Percona XtraDB Cluster
```

Comment:
Why to include this file if the release 8.0.38 is skipped? Why not give this info in preamble of 8.0.39?

Reply:
because MySQL released 8.0.38 so users can search for it in theory.

Reply:
It's the same pattern we used for PS, PDPS (PS), and PXC.